services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -778,6 +778,10 @@ input ContentHasWebsiteScheduleInput {
   sectionBubbling: Boolean = true
 }
 
+input SectionStartDateInput {
+  sectionId: Int
```

Comment:
Add GraphQL docs here to indicate what the input is used for and what each field (including the new flag) does